validator/lib/validateTileset.js

```diff
@@ -168,3 +191,38 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function checkBoundingVolume(tileBV, parentBV, tileTransform, parentTransform, type) {
```

Comment:
So for this case you mean wherever we do those checks like 'regionInsideRegion' and such in the above code it should be replaced with 'checkBoundingVolume' and just take in the transforms and the bounding volumes and based on the type of the bounding volume operate on it?